en/02_Developer_Guides/06_Testing/index.md

```diff
@@ -55,25 +55,26 @@ vendor/bin/phpunit vendor/silverstripe/framework/tests '' flush=1
 PHPUnit can generate a code coverage report ([docs](http://www.phpunit.de/manual/current/en/code-coverage-analysis.html))
 which shows you how much of your logic is executed by your tests. This is very useful to determine gaps in tests.
 
-	:::bash
-	vendor/bin/phpunit --coverage-html <output-folder> <optional-tests-folder>
+```
```

Comment:
```suggestion
```bash
```

Reply:
Updated both occurrences of bash code blocks